build.gradle

```diff
@@ -120,6 +120,18 @@ dependencyManagement {
         dependency group: 'commons-beanutils', name: 'commons-beanutils', version: versions.commonsBeanUtils
         // CVE-2018-10237
         dependency group: 'com.google.guava', name: 'guava', version: '30.1-jre'
+        // CVE-2020-26939
```

Comment:
If you have updated the vulnerable version of this package, dont forget to remove it from the suppressions file!